### DISCUSSION TITLE
Add this module to Nuxt modules

### DISCUSSION THREAD

**Discussion by tomsq** [ivodolenc/nuxt-font-loader#24] [Ideas]
### Request Description

I've found this module to be really helpful. It helped me to improve page perf quite a lot. Have you thought about adding this module to the Nuxt modules for better discoverability?

### Alternative Solutions

-

**Comment by ivodolenc** [comment]
Hi, I'm glad you found the module useful.

> Have you thought about adding this module to the Nuxt modules for better discoverability?

Not at the moment, but feel free to suggest the module to their official modules repo 👍 Definitely, this would help others find the module more easily.
